editions/tw5.com/tiddlers/saving/Saving.tid

```diff
@@ -26,16 +26,13 @@ type: text/vnd.tiddlywiki
 \end
 
 \define uncheckactions(item:"Linux")
-<$action-listops  $subfilter="-[[$item$]]"/>
+<$action-listops  $tiddler=<<stateTiddler>> $subfilter="-[[$item$]]"/>
 \end
 
 \define checkactions(item:"Linux")
-<$action-listops $subfilter="[[$item$]]"/>
-\end
-
-\define uncheckactions(item:"Linux")
-<$action-listops $subfilter="-[[$item$]]"/>
+<$action-listops $tiddler=<<stateTiddler>>  $subfilter="[[$item$]]"/>
 \end
+<$vars stateTiddler=<<qualify "$:/state/">> >
```

Comment:
It would be better for debugging to use a prefix like `$:/state/gettingstarted`

Reply:
Changed tiddler accordingly.